book/doctrine.rst

```diff
@@ -979,126 +1060,66 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
             </entity>
         </doctrine-mapping>
 
-First, since a ``Category`` object will relate to many ``Product`` objects,
-a ``products`` array property is added to hold those ``Product`` objects.
-Again, this isn't done because Doctrine needs it, but instead because it
-makes sense in the application for each ``Category`` to hold an array of
-``Product`` objects.
+Doctrine does not *require* that the "one" side of a one-to-many relationship
+hold a collection of its "many" related entities. But in the context of our
```

Comment:
Not to split hairs, but using the present subjunctive is grammatically appropriate here.